<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeUnitItemsFormClass</class>
 <widget class="QDialog" name="ChangeUnitItemsFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeUnitItemsForm</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>781</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="3" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Applied items</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" rowspan="8">
     <widget class="QListWidget" name="AppliedItems">
      <property name="minimumSize">
       <size>
        <width>478</width>
        <height>356</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Items</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" rowspan="8" colspan="2">
     <widget class="QListWidget" name="ItemsListWidget"/>
    </item>
    <item row="8" column="4">
     <widget class="QPushButton" name="RemoveItemButton">
      <property name="text">
       <string>Remove Item</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="ToMenu">
      <property name="styleSheet">
       <string notr="true">background-color:#F0FFFF;
font-weight:800;
color:black;
</string>
      </property>
      <property name="text">
       <string>Go back</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="ApplyItemButton">
      <property name="text">
       <string>Apply Item</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
